### ISSUE TITLE
rtpengine-recording and mysql

### ISSUE THREAD

**Issue by john08burke** [sipwise/rtpengine#629]
Hello:

I am running rtpengine and recording-daemon (mr6.4.1.1) on debian stretch. I'm using latest version of mariadb to store metadata key/val pairs. However, I noticed a huge amount of UPDATES to the db while calls are up (~100/sec/call).

Logging the db queries it looks like the culprit is the update recording_streams query.
`update recording_streams set channels = ?, sample_rate = ? where id = ?;`

See attached logs:
[mariadb.log](https://github.com/sipwise/rtpengine/files/2447455/mariadb.log)

rtpengine-recording.conf
```
[rtpengine-recording]
table = 0
num-threads = 16
output-storage = file
output-format = mp3
output-mixed = 1
spool-dir = /var/spool/rtpengine
output-dir = /var/lib/rtpengine-recording

### mysql configuration for db storage
mysql-host = localhost
mysql-port = 3306
mysql-user = fakeusr
mysql-pass = fakepwd
mysql-db = rtpengine
```

rtpengine.conf
```
[rtpengine]
table = 0
interface = internal/192.168.20.47
listen-cli = 9900

listen-ng = 192.168.20.47:22222
listen-udp = 12222

timeout = 60
silent-timeout = 3600
tos = 184

pidfile = /var/run/ngcp-rtpengine-daemon.pid
num-threads = 16

port-min = 55000
port-max = 60000

recording-dir = /var/spool/rtpengine
recording-method = proc
```
Any help would be appreciated!
Thanks,
John

**Comment by juha-h** [comment]
I noticed the same.  Is this a bug in intended behavior?

**Comment by juha-h** [comment]
Why would number of channels and sample rate need to be updated that often?  Normally they do not change during the call at all.

**Comment by juha-h** [comment]
While waiting for a proper fix, I got rid of the syslog messages by commenting out db_config_stream(output); call in output.c.  The only side effect is that channels and sample_rate columns are null in recording_streams table.

**Comment by rfuchs** [comment]
The fix should be as simple as e300684. Can you confirm?

**Comment by juha-h** [comment]
Richard Fuchs writes:

> The fix should be as simple as e300684. Can you confirm?

I tested and the fix worked.  If possible, cherry-pick to mr6.5.1 or to
a new branch would be nice.

**Comment by john08burke** [comment]
Works! Thanks.
